rules/go_rules.build_defs

```diff
@@ -1026,8 +1025,13 @@ def _go_github_repo_cmd(name, get, repo, revision, module_major_version):
 
 def _set_go_env():
     if CONFIG.GOROOT:
-        return f'export GOPATH=$TMP_DIR && export GOROOT={CONFIG.GOROOT}'
-    return 'export GOPATH=$TMP_DIR && eval $("$TOOLS_GO" env)'
+        go_root = CONFIG.GOROOT
+    elif CONFIG.GO_TOOL.startswith(":") or CONFIG.GO_TOOL.startswith("//"):
+        label = CONFIG.GO_TOOL.split('|')[0]
```

Comment:
nit: I might write this as `label, _ = CONFIG.GO_TOOL.rpartition('|')`

I have an irrational hatred for following up parentheses with square brackets that way ;-)

Reply:
o.O okay! That's much nicer... sorry I don't know how to python